lldb/include/lldb/Interpreter/CommandObjectMultiword.h

```diff
@@ -70,6 +70,8 @@ class CommandObjectMultiword : public CommandObject {
     return m_subcommand_dict;
   }
 
+  std::string GetTopSubcommands(int count);
```

Comment:
Can we call this GetSubcommandsUpTo(int count) or something like that.  "TopSubcommands" sounds like you are returning the most popular ones.

Also, maybe size_t not int since this can't be negative.

Reply:
This is definitely a topic for another patch, but I actually think it would be really cool if this actually returned the first N most popular commands. :) Then we could even have `help breakpoint` start off with the most popular subcommand (definitely `set`) instead of `clear` (which I don't think I've ever used).

Reply:
Will rename this to Jim's suggestion.
I like Pavel's idea of having this returning the popular ones, but not sure how we'd determine that here (other than hard-coding the list somewhere)

Reply:
It would have to be really clear that's what we were doing, however.  If you saw this list come out, then were curious and did `help breakpoint` and the list came out in a different order that would just look like we were being weird.

Reply:
Yeah, I was basically thinking just some predetermined order (our estimate of a command usage). I don't want to derail this patch with it though, I just found the remark humorous and couldn't resist the temptation.

Reply:
P.S: Actually, didn't end up renaming it Jim's suggestion, but to something else, hopefully less confusing, which is "GetSubcommandsHint" - whether it returns a top 5 sub-commands or whatever order is transparent to caller.
(Then we can change the content in later patch as needed)

Reply:
Sure, that's fine too.